pandas_gbq/gbq.py

```diff
@@ -411,7 +411,20 @@ def run_query(self, query, max_results=None, progress_bar_type=None, **kwargs):
         timeout_ms = job_config_dict.get("jobTimeoutMs") or job_config_dict[
             "query"
         ].get("timeoutMs")
-        timeout_ms = int(timeout_ms) if timeout_ms else None
+
+        if timeout_ms:
+            timeout_ms = int(timeout_ms)
+            # Having too small a timeout_ms results in individual
+            # API calls timing out before they can finish.
+            # ~300 milliseconds is rule of thumb for bare minimum
+            # latency from the BigQuery API.
+            minimum_latency = 400
```

Comment:
Maybe we can define `minimum_latency` as a constant at top of the file, for easy access and reuse in the future (see [example](https://github.com/googleapis/python-bigquery/blob/0e390668aaa411d59cc99514c89da8b23af5789a/google/cloud/bigquery/client.py#L136-L137)).

Reply:
I would like to leave this as is.

Reply:
Could you explain why? Thank you.

Reply:
@Linchin 

I don't see this particular variable being used widely throughout the code (now or in the future).

I appreciate having the variable and the comment about the variable close together and in this case close to where they are being used. I feel it improves the readability/maintainability of the code. Especially since this edit is in response to a flakybot failing test. If it continues to fail, then whoever troubleshoots this is likely to see the comment in situ and may then tweak the duration to better match real world conditions.